app/Check/Commands/SlackCheckStatusesCommand.php

```diff
@@ -130,7 +137,17 @@ protected function execute(
 				$checkStatusMessage ? ': ' . $checkStatusMessage : ''
 			);
 
-			$this->slackNotifier->notify('#monitoring', $message, $color);
+			if ($check->project->notifications) {
+				$this->slackNotifier->notify('#monitoring', $message, $color);
+			}
+
+			foreach ($check->project->userSlackNotifications as $user) {
+				$identity = $this->userRepository->getById($user->user->getId());
```

Comment:
Ta identita je jednodušší: `$identity = $user->user;`. IMHO bych si ji ani nevytahoval, ale rovnou bych se na to doptal v místě použití, ale to nechám na tobě.